gpkit/__init__.py

```diff
@@ -156,3 +156,4 @@ def load_settings(path=SETTINGS_PATH):
         settings_ = {"installed_solvers": [""]}
     return settings_
 settings = load_settings()
+settings["latex_modelname"] = False
```

Comment:
should we make the default `settings["latex_modelname"] = False`, so that this doesn't affect the status quo? I'm willing to make the settings change for my applications where I don't want printing, and I like the idea of leaving printing on to remind us that a longer-term fix is still needed.